infra/util/transform_file_to_string_template.cpp.conf

```diff
@@ -0,0 +1,12 @@
+#include "${name}.hpp"
+#include <array>
+
+namespace ${namespace}
```

Comment:
Would it make sense to prepend the namespace with some fixed namespace?

```c++
namespace emil::generated::${namespace}
{
    ...
}
```
?

Reply:
You have full control over the namespace, so we leave that up to the client.